packages/packages.json

```diff
@@ -1 +1,96 @@
-{}
\ No newline at end of file
+{
```

Comment:
@solarw Are you sure you need all of these packages?

Reply:
@angrybayblade used copy of all packages for the development (package inspection, autocompletition)
so packages were indexed.

Reply:
@angrybayblade cleaned up packages. but some packages are still required as a local copy, otherwise dpeloyment does not start